source/ios/tools/IOSFeed/IOSFeed.cs

```diff
@@ -0,0 +1,146 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Blob;
+
+namespace AdaptiveCards.Tools.IOSFeed
+{
+    public static class Constants
+    {
+        public const string ConnectionStringPath = "source/ios/tools/IOSFeed/ConnectString.txt";
```

Comment:
> = "source/ios/tools/IOSFeed/ConnectString.txt" [](start = 48, length = 47)

This is PII -- is this assumed to be in clear text?

Reply:
AzureDepOp has secure file download task. The task stores a file securely, then it download the file securely. And the location is referenced by a variable specified by the task.

unless build machine is hacked during the build process, this practice won't pose security risk.

---
In reply to: [314079827](https://github.com/microsoft/AdaptiveCards/pull/3291#discussion_r314079827) [](ancestors = 314079827)